cmd/bblfsh-tools/common.go

```diff
@@ -0,0 +1,73 @@
+package main
+
+import (
+	"context"
+	"io/ioutil"
+
+	"google.golang.org/grpc"
+
+	"github.com/Sirupsen/logrus"
+	"github.com/bblfsh/sdk/protocol"
+	"github.com/bblfsh/sdk/uast"
+	"github.com/bblfsh/tools/tools"
+	errors "srcd.works/go-errors.v0"
```

Comment:
No need to add an alias here. Even if the repo is named `go-errors.v0`, the package is named `errors`. It is quite confusing, but package names on imports are based on the `package` declaration at the beginning of files, not the URL.